packages/server-side-render/README.md

```diff
@@ -48,6 +48,12 @@ Examples: "my-custom-server-side-rendered".
 - Type: `String`
 - Required: No
 
+### requestBody
```

Comment:
IMO `requestBody` is not a good name for this. It's not clear that this is a `bool` flag. I would expect `requestBody` to allow me to specify the actual request body.

Why not rename it to something like `httpMethod`? It would allow `GET` or `POST`.

Alternatively, `requestBody` could accept an object. If I pass `requestBody={{ attributes }}`, `method` and `path` would be handled accordingly,  and the `attributes` prop would be ignored.

Reply:
Thanks, @swissspidy! I'll rename `requestBody` to `httpMethod`.

Reply:
Thanks, that's a good suggestion. It's applied in https://github.com/WordPress/gutenberg/compare/6eb0b0803aa53ca6001f9afdc97f88857d364eab...42153114ea71e2690a83fdba1317f268a986e4ac